File: Lista_Geral_Python/comandos de entrada e saida/question_9.py
###  Escreva um programa que leia o valor de uma distância percorrida em km e o tempo 
# gasto em horas e calcule a velocidade média em m/s.

distancia = int(input("Informe a distancia percorrida: "))
tempo = int(input(f"Informe o tempo em que percorreu {distancia}km: "))
print(f"A distacia percorrida foi: {distancia}km\nFoi percorrida em: {tempo}h\nEm uma velocidade de {(distancia / tempo):.1f}km/h\nEm m/s fica: {((distancia / tempo) / 3.6):.2f}m/s")

File: Lista_Geral_C/comandos de entrada e saida/question9.c
//Escreva um programa que leia o valor de uma distância percorrida em km e o tempo 
//  gasto em horas e calcule a velocidade média em m/s.
#include <stdio.h>

int main(){

    float distancia, tempo;
    float velocidade_media;

    printf("Informe a distancia percorrida: ");
    scanf("%d", &distancia);
    printf("Informe o tempo em que foi percorrido: ");
    scanf("%d",&tempo);

    velocidade_media = (distancia / tempo) / 3.6;

    printf("A velocidade media foi: %1.f m/s.\n",velocidade_media);


    return 0;
}